WordPress/Classes/ViewRelated/Menus/Views/MenuItemSourceCell.m

```diff
@@ -0,0 +1,165 @@
+#import "MenuItemSourceCell.h"
+#import "WPStyleGuide.h"
+#import "WPFontManager.h"
+#import "Menu+ViewDesign.h"
+
+#pragma mark - MenuItemSourceRadioButton
+
+@interface MenuItemSourceRadioButton : UIView
+
+@property (nonatomic, assign) BOOL selected;
+@property (nonatomic, assign) BOOL drawsHighlighted;
+
+@end
+
+#pragma mark - MenuItemSourceOptionView
+
+static CGFloat const MenuItemSourceCellHierarchyIdentationWidth = 17.0;
+
+@interface MenuItemSourceCell ()
+
+@property (nonatomic, strong, readonly) UIStackView *stackView;
+@property (nonatomic, strong, readonly) UIStackView *labelsStackView;
+@property (nonatomic, strong, readonly) UILabel *label;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *leadingLayoutConstraintForContentViewIndentation;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *topLayoutConstraintForContentViewIndentation;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *topLayoutDefaultConstraint;
+
+@end
+
+@implementation MenuItemSourceCell
+
+- (void)dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
+{
+    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
+    if (self) {
+        
+        self.backgroundColor = [UIColor whiteColor];
+        
+        [self initStackView];
+        [self initLabelsStackView];
+        [self initLabel];
```

Comment:
I think I agree there, we added these as separate `init` method in https://github.com/wordpress-mobile/WordPress-iOS/pull/5242. But the more I've looked at them the more something felt off, I think the use of `init` as you mentioned is the issue. I'll switch these around to `setup` prefixes instead.